meta/templatetags/meta.py

```diff
@@ -2,11 +2,43 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from django import template
+from django.conf import settings
 from django.utils.html import escape
+from django.utils.safestring import mark_safe
 from django.utils.six import string_types
 
 register = template.Library()
 
+try:
+    from django.apps import apps
```

Comment:
django.apps only exists in Django 1.7+, this package currently supports also Django 1.6